cdap-ui/app/cdap/components/DataPrep/Directives/FindAndReplace/index.js

```diff
@@ -77,9 +92,16 @@ export default class FindAndReplaceDirective extends Component {
     let column = this.props.column;
     let findInput = this.state.findInput;
     let replaceInput = this.state.replaceInput;
+    if (this.state.exactMatch) {
+      findInput = `^${findInput}$`;
+    }
+    let directive = `find-and-replace ${column} s/${findInput}/${replaceInput}`;
 
-    let directive = `find-and-replace ${column} s/${findInput}/${replaceInput}/g`;
-
+    if (this.state.ignoreCase) {
+      directive = `${directive}/Ig`;
```

Comment:
No. Thats what I thought too. But apparently `I` is used instead of `i` for case insensitive search and replace